main.go

```diff
@@ -108,36 +108,41 @@ func main() {
 					return
 				}
 
-				// validate the cluster ids contained in the remote write request
-				clusterId, err := remotewrite.ValidateRequest(remoteWriteRequest)
-				if err != nil {
-					log.Printf("error validating remote write request: %v", err)
-					w.WriteHeader(http.StatusForbidden)
-					return
-				}
+				clusterId := ""
+				if !remotewrite.IsMetadataRequest(remoteWriteRequest) {
+					// validate the cluster ids contained in the remote write request
+					clusterId, err = remotewrite.ValidateRequest(remoteWriteRequest)
+					if err != nil {
+						log.Printf("error validating remote write request: %v", err)
```

Comment:
Let's quote the clusterID in the logs to make sure we see it is empty. This should be done for all the other logs that log it

Reply:
This is not really needed in this specific case as I understand it should not be empty. I leave it up to you to decide if we want to be consistent with it or not